src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -321,6 +321,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
     <UpToDateCheckBuilt Include="@(RazorIntermediateAssembly)"
       Condition="'$(RazorCompileOnBuild)'=='true' AND '@(Content->WithMetadataValue('Extension', '.cshtml'))' != ''" />
+
+    <!-- Set up watchers for dotnet-watch -->
+    <Watch Include="@(Content->WithMetadataValue('Extension', '.razor'))" />
```

Comment:
Also adding support for dotnet-watch's file list.

Reply:
Is there value to adding a condition flag for this?

Reply:
If we end up requiring it, I'd want to try and add a feature to dotnet-watch (expose a target). Right now, it's fairly unusual scenario to not want to rebuild if .razor files change